src/common/Errors.ts

```diff
@@ -33,6 +24,34 @@ class _Errors {
 		);
 	}
 
+	onStorageOptionsChange = (data: StorageOptionsChangeData) => {
+		if (data.options && 'allowRollbar' in data.options) {
+			this.checkRollbar();
+		}
+	};
+
+	checkRollbar() {
+		const { allowRollbar } = BrowserStorage.options;
+		if (allowRollbar && !this.rollbar) {
+			this.rollbar = new Rollbar({
+				accessToken: secrets.rollbarToken,
+				autoInstrument: {
+					network: false, // Do not set to true on Firefox (see https://github.com/rollbar/rollbar.js/issues/638).
+				},
+				captureIp: false,
+				captureUncaught: true,
+				captureUnhandledRejections: true,
+				payload: {
+					environment: 'production',
```

Comment:
I think it would be better to read this from an enviroment variable. So that the same rollbarToken could be used for both production and development. (Of course not important improvement.)